drivers/hmis/lib/hmis_util/hud_form_rules_2024.rb

```diff
@@ -0,0 +1,333 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisUtil
+  class HudFormRules2024
+    # Keys match Link IDs in our Form Definition fragments for HUD data elements
+    HUD_LINK_ID_RULES = {
+      # CoC Code for Client Location
+      q_3_16: {
+        stages: [:INTAKE],
+        data_collected_about: :HOH,
+      },
+      # Prior Living Situation A
+      q_3_917A: {
```

Comment:
It works, but it feels like an anti-pattern to rely on a certain link id outside of the form logic. The alternative would be to add some new attribute like `"hud_data_element_id": "3.917"` and do the checks based off that.